docs/clusterdefinition.md

```diff
@@ -26,6 +28,11 @@ Here are the valid values for the orchestrator types:
 4.  `Swarm Mode` - this represents the [Swarm Mode orchestrator](swarmmode.md).
 5.  `OpenShift` - this represents the [OpenShift orchestrator](openshift.md).
 
+
+Examples for valid values for the orchestrator release: `1.9`, `1.10`, `1.11`, etc.
+
+Examples for valid values for the orchestrator version: `1.9.10`, `1.10.7`, `1.11.2`, etc.
```

Comment:
this is only true for `Kubernetes` and supported versions could change overtime. Instead of listing a few hardcoded versions, how about pointing people to the `orchestrators` command which gives info on supported versions such as:

To learn more about supported orchestrators and versions, run the *orchestrators* command.
```bash
/bin/acs-engine orchestrators
```